README.md

```diff
@@ -24,4 +24,4 @@ Those, who prefer coding, please see the [contributing guidelines](CONTRIBUTING.
 
 ## Privacy policy
 
-The privacy policy is available on the [wiki](https://github.com/MikkCZ/pontoon-addon/wiki/Privacy-Policy) and apply to both Firefox and Chrome versions of Pontoon Add-on.
+The privacy policy is available [here](privacy.md) and apply to both Firefox and Chrome versions of Pontoon Add-on.
```

Comment:
Oh, maybe PRIVACY.md in all capitals, so it's clear on first sight that it's a text file, not part of the code (yet)?